Header header
float64 Latitude
float64 Longitude
float64 UTM_northing
float64 UTM_easting
float64 Altitude
int32 Zone
string Letter

